tetris.js

```diff
@@ -412,5 +412,23 @@ module.exports = {
 		let newCoordinates = this.rotatesArrayOfCoordinates(tetrominoCoordinates, pivot)
 		let result = this.shiftCoordinates(newCoordinates, shift)
 		return result.sort(this.orderCoordinates);
+	},
+	incrementPhase: function (typeOfTetromino, currentRotationPhaseOfTetromino) {
+		let arrayPhases = [];
+		if(typeOfTetromino === "I" || typeOfTetromino === "S" || typeOfTetromino === "Z"){
+			arrayPhases = ["vertical","horizontal"];
+		}
+		if(typeOfTetromino === "L" || typeOfTetromino === "J" || typeOfTetromino === "T"){
+			arrayPhases = ["up","left","down","right"];
+		}
+		if(typeOfTetromino === "O"){
+			arrayPhases = ["stable"];
+		}
+		let value = arrayPhases.indexOf(currentRotationPhaseOfTetromino)
+		if(arrayPhases.length-1 > value){
```

Comment:
Please use equality "===" check here instead of less then/more than check
Also value should be on the left and constant should be on the right if comparison, by convention